src/chrome/content/rules/Intel.xml

```diff
@@ -58,12 +54,14 @@ Fetch error: http://scoop.intel.com/ => https://scoop.intel.com/: (7, 'Failed to
 	Problematic subdomains:
 		- chi1      (Expired)
 		- embedded.edc ³
+		- firmware ⁴
```

Comment:
It's not a mismatch, it is a issue in the certificate chain. And the target is still active.